src/js/angular/aclmanagement/controllers.js

```diff
@@ -20,6 +20,12 @@ function AclManagementCtrl($scope, toastr, AclManagementRestService, $repositori
     // Public fields
     //
 
+    /**
+     * The model for the rule editing form.
+     * @type {{}}
```

Comment:
Is that `object`?

Reply:
uhmmm yeah. It was autogenerated by the IDE and I didn't paid attention. It's the angular's form model bound to this controller's scope.